lib/chord-metadata/docker-compose.yml

```diff
@@ -13,61 +13,30 @@ services:
     depends_on:
       - metadata-db
     environment:
-      - CHORD_URL=${CHORD_METADATA_HOST}
-      - CHORD_PERMISSIONS=${CHORD_METADATA_AUTH}
-      - CHORD_DEBUG=${CHORD_METADATA_DEBUG}
       - POSTGRES_HOST=metadata-db
+      - POSTGRES_PORT=5432
       - POSTGRES_USER=admin
       - POSTGRES_PASSWORD_FILE=/run/secrets/metadata_db_secret
-      - INSIDE_CANDIG=${INSIDE_CANDIG}
       - OPA_URL=${OPA_PRIVATE_URL}
-      - CANDIG_AUTHORIZATION=${CANDIG_AUTHORIZATION}
-      - CACHE_TIME=${CACHE_TIME}
+      - KATSU_AUTHORIZATION=${CANDIG_AUTHORIZATION}
       - OPA_SITE_ADMIN_KEY=${OPA_SITE_ADMIN_KEY}
       - HOST_CONTAINER_NAME=${CHORD_METADATA_CONTAINER}
     secrets:
-      - source: metadata-app-secret
-        target: metadata_app_secret
-      - source: metadata-db-user
-        target: metadata_db_user
       - source: metadata-db-secret
         target: metadata_db_secret
       - source: opa-root-token
         target: opa-root-token
-    entrypoint:
```

Comment:
This deletion means that this PR shouldn't be merged until [moh_part_24](https://github.com/CanDIG/katsu/pull/102) is merged. Otherwise this looks good and works for me.